naywatch/files/naywatch.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+
+. /lib/functions.sh
+. /lib/functions/network.sh
+
+CHECK_INTERVAL=$1
+shift
+WATCHDOG_TIMEOUT=$1
+shift
+USE_WATCHDOG=$1
+shift
+SAVE_LOGS=$1
+shift
+INTERFACES="$*"
+
+ACTIVE=0
+
+log() {
+    local msg="$1"
+    logger -t naywatch "$msg"
+}
+
+write_logs() {
+    save_log() {
+        eval $1 > /root/$(date +%s)-"$1".log
+    }
+    config_load naywatch
+    config_list_foreach general save_cmd save_log
+    sync
+}
+
+neighbors_available() {
+    for interface in $INTERFACES; do
+        network_get_physdev phy $interface > /dev/null 2>&1
+        linklocal=$(ip -6 a list dev $phy | grep "scope link" | awk '{print $2}' | sed 's/\/64//') 2> /dev/null
+        ips=$(ping ff02::1%$phy -c2 | grep from | awk '{print $4}' | sed 's/.$//') 2> /dev/null
```

Comment:
This could be done shorter.
`grep from | awk '{print $4}'`
=>
`awk '/from/{print($4)}'`
(same for the other calls)
The sed call can be included using `awk -F '[. ]+' ...`).

The ping calls might benefit from some timeout/deadline (-W, -w).

Reply:
I tried
```
 ping ff02::1%eth0.42 -w5 -W5 -c2 | awk -F '[. ]+' '/from/{p
rint($4)}'
```
But this awk -F does not work? Just searching for some documentation

Reply:
-F sets the regex that is used to separate the field (usually whitespace).
I cannot test right now. Feel free to use or not. :)